pwiz/analysis/spectrum_processing/SpectrumList_ChargeFromIsotope.cpp

```diff
@@ -919,6 +945,32 @@ void SpectrumList_ChargeFromIsotope::getParentPeaks(const SpectrumPtr s,const ve
                 peakMZs.erase( peakMZs.begin() + elementsForDeletion[j] - j );
             }

```

Comment:
Hmm, I don't recall why we didn't use vendor centroiding when available. Maybe because I was supremely confident in the abilities CantWaiT! I can't think of a technical reason why that would be a bad idea. It might be that for my paper we wanted to avoid mixing and matching our new methods with vendor-provided algorithms.